spec/javascripts/health-care/components/questions/FullName.spec.jsx

```diff
@@ -21,60 +52,28 @@ describe('<FullName>', () => {
     expect(selects).to.have.length(1);
   });
 
-  it('sets and removes error css on blank name', () => {
-    // Initial state should be valid.
-    expect(component.state.hasError).to.be.false;
-    expect(ReactTestUtils.scryRenderedDOMComponentsWithClass(
-        component, 'usa-input-error')).to.have.length(0);
-
-    // Empty first and last name renders error.
-    component.refs.first.value = '';
-    component.refs.last.value = '';
-    ReactTestUtils.Simulate.change(component.refs.last);
-    expect(component.state.hasError).to.be.true;
-    expect(ReactTestUtils.scryRenderedDOMComponentsWithClass(
-        component, 'usa-input-error')).to.have.length(2);
-
-    // Present first and last name removes error.
-    component.refs.first.value = 'a';
-    component.refs.last.value = 'b';
-    ReactTestUtils.Simulate.change(component.refs.last);
-    expect(component.state.hasError).to.be.false;
-    expect(ReactTestUtils.scryRenderedDOMComponentsWithClass(
-        component, 'usa-input-error')).to.have.length(0);
+  it('excludes ErrorMessage prop when valid name', () => {
+    const tree = SkinDeep.shallowRender(<FullName value={{ first: 'First', middle: '', last: 'Last', suffix: '' }} onUserInput={(_update) => {}}/>);
+    const errorableInputs = tree.everySubTree('ErrorableTextInput');
+    expect(errorableInputs).to.have.lengthOf(3);
+    expect(errorableInputs[0].props.errorMessage).to.be.undefined;
   });
 
-  it('validate first and last name are present', () => {
-    // Present first name and blank last name should render error.
-    component.refs.first.value = 'a';
-    component.refs.last.value = '';
-    ReactTestUtils.Simulate.change(component.refs.last);
-    expect(component.state.hasError).to.be.true;
-
-    // Present first and last should have no error.
-    component.refs.first.value = 'a';
-    component.refs.last.value = 'b';
-    ReactTestUtils.Simulate.change(component.refs.last);
-    expect(component.state.hasError).to.be.false;
-
-    // Blank first and last should render error.
-    component.refs.first.value = '';
-    component.refs.last.value = '';
-    ReactTestUtils.Simulate.change(component.refs.last);
-    expect(component.state.hasError).to.be.true;
-
-    // Blank middle name should not render error.
-    component.refs.first.value = 'a';
-    component.refs.middle.value = '';
-    component.refs.last.value = 'b';
-    ReactTestUtils.Simulate.change(component.refs.last);
-    expect(component.state.hasError).to.be.false;
+  it('includes ErrorMessage prop when blank name', () => {
```

Comment:
I added these 3 tests like you recommended, but for some reason this test is failing. It says: `AssertionError: expected undefined not to be undefined`. Not sure why `ErrorMessage` isn't being set for `first` and `last`. Tried stepping through the test in the debugger and couldn't figure it out.